src/components/TraceDetail.tsx

```diff
@@ -15,6 +15,10 @@ import type { QueryInfo as TraceDetailProps } from './TraceViewerPanel';
 import { SpanOverlayDrawer } from './Span/SpanOverlayDrawer';
 import { HelpModal } from './HelpModal';
 
+// default Grafana does not support child count.
+// In production, we use a custom build of Grafana that supports child count.
+const supportsChildCount = false;
```

Comment:
Not sure, how to best deal with this.
Updating it to true in production works for now.
Maybe there is a webpack trick we can apply?